.github/workflows/push.yml

```diff
@@ -79,9 +79,10 @@ jobs:
     needs:
       - set_version
     concurrency:
-      group: tests
+      group: tests-${{ matrix.name }}
```

Comment:
Dnes nejak extremne padaju `Github Actions`. Same `500`ky.

`concurrency` je beta feature, ktoru pouzivame na to, aby nebezalo oproti KBC testovacim projektom viacero pipelines paralelne. Vramci pipeline je synchronizacia vyriesena, ale ak sa spusti viac CI pipelines, tak o sebe nevedia. `concurrency` ma tomu zabranit.

Je znamy bug, ze aj ked je nastavene `cancel-in-progress: false`, tak nova pipelina zabije vsetky `pending` tasky (na tie `in-progess` caka: https://github.com/github/feedback/discussions/5435)

No dnes sa mi zacali navzajom zabijat tasky aj vramci jednej pipeline.
Pouziva sa tu `matrix`, tj. je to jeden step, ktory bezi paralelne pre `linux,mac,win`.
V dokumentacii nemaju napisane ako v tomto pripade nastavit `concurrency`.

Mam podozrenie, ze by mohlo pomoct pridat k nazvu zamku nazov operacneho systemu, ... takze kazdy job z `matrix`, ktory ma bezat paralelne ma svoj zamok.